### DISCUSSION TITLE
Apple Authentication

### DISCUSSION THREAD

**Discussion by chrisdhanaraj** [supabase/supabase#1015] [Questions]
Hello there! I'm exploring Supabase for a project and I was trying to see if Apple authentication was on the roadmap somewhere? I'm shipping a mobile app to the various app stores, and, well, iOS now requires Apple Auth if you expose the other social providers :(

**Comment by steve-chavez** [comment]
Hey @chrisdhanaraj,

Yes, Apple auth is being tracked on https://github.com/supabase/gotrue/issues/37.

**Reply by chrisdhanaraj** [reply]
Ah bummer :( I'll check back in once Apple Auth gets going! Excited to use this
